src/main/java/mt/edu/um/cf2/jgribx/grib2/Grib2RecordPDS.java

```diff
@@ -110,8 +110,8 @@ public Grib2RecordPDS(GribInputStream in, ProductDiscipline discipline, Calendar
                 parameter = Grib2Parameter.getParameter(discipline, paramCategory, paramNumber);
                 if (parameter == null)
                 {
-                    throw new NotSupportedException("Unsupported parameter: D:" + discipline + " C:" + paramCategory
-                            + " N:" + paramNumber);
+                    Logger.println("Unsupported parameter: D:" + discipline + " C:" + paramCategory
```

Comment:
Not sure I agree with this change. If the parameter is not recognized, I want to skip the entire GRIB record since I don't want to keep records with unknown data. What do you think?

Reply:
You are probably right, just it probably it needs to skip proper amount of data, otherwise it does not work.